### PR TITLE
Fix: Exceptions when executing via QuantumInstance

### LINKED ISSUE #21: Executing via QuantumInstance throws exceptions
Specifically, when running with `get_backend` (running against a real QPU), it complains about our configuration object not having `max_experiments` set.

@zohimchandani reported this, we'll want to wait for a reservation to confirm the fix (and check for any other issues).

### PR DESCRIPTION
**Pull request by dbanty**
Fixes #21. There may be more issue as well, so this is a draft until I can pair with @zohimchandani during a reservation to test the fix.

@rigetti/qcs `max_experiments` is how many experiments can be submitted as a single job (basically, how many we'll submit before waiting on all the results). I'm not sure what a good number is here, I don't believe we have any equivalent in pyQuil. Thoughts?

### CODE REVIEW

**Comment by mhodson-rigetti** [comment]
@dbanty when we're using pyquil v3's asynch job submission under the hood, 8 jobs in parallel has shown to be a good maximum, above which there isn't really any further throughput increase.

**Comment by dbanty** [comment]
Thanks @mhodson-rigetti! I'll adjust the upper limit to 8 for now then.

File: qiskit_rigetti/_qcs_provider.py [hunk 1]
```diff
@@ -127,4 +127,5 @@ def _configuration(name: str, num_qubits: int, local: bool, simulator: bool) ->
         memory=False,
         max_shots=10000,
         coupling_map=[],
+        max_experiments=100,
```
Comment (dbanty) [inline qiskit_rigetti/_qcs_provider.py:130]:
```suggestion
        max_experiments=8,
```
Setting to 8 per @mhodson-rigetti's recommendation.

**Comment by dbanty** [comment]
I got word from @zohimchandani that this fix does the trick and the Qiskit code in question can run now!

**Review by notmgsk — APPROVED** [review]

**Pull request merged**